symphony/bdk/core/client/api_client_factory.py

```diff
@@ -69,7 +69,7 @@ async def close_clients(self):
         await self._session_auth_client.close()
 
     def _get_api_client(self, server_config, context) -> ApiClient:
-        configuration = Configuration(host=(server_config.get_base_path() + context))
+        configuration = Configuration(host=(server_config.get_base_path() + context), discard_unknown_keys=True)
```

Comment:
I had already created a no pointer task to add this: [PLAT-10751](https://perzoinc.atlassian.net/browse/PLAT-10751)
Will close it as fixed by this one.

Reply:
Yes, because datafeed example was failing because of that (new stateful flag added for VSM).